js/dashboard.js

```diff
@@ -56,14 +63,7 @@
 		],
 	} );
 
-	pointRadius = 4;
-	if ( data.length > 365 ) {
-		pointRadius = 0;
-	} else if ( data.length > 180 ) {
-		pointRadius = 1;
-	} else if ( data.length > 90 ) {
-		pointRadius = 2;
-	}
+	pointRadius = 2;
```

Comment:
For displaying only a small dataset (e. g. the last 14 days) only I like the display with pointRadius = 4. I think this code should only be changed for the `data.length > 365` case (due to the tooltip not being shown if there's no point to hover)